inc/oapv_version.h.in

```diff
@@ -47,4 +48,9 @@
     ((major) == OAPV_LIB_VERSION_MAJOR && (minor) < OAPV_LIB_VERSION_MINOR) || \
     ((major) == OAPV_LIB_VERSION_MAJOR && (minor) == OAPV_LIB_VERSION_MINOR && (patch) <= OAPV_LIB_VERSION_PATCH))
 
-#endif /* __OAPV_LIBVERSION_H__8465216516551324987461894651851187416502317561968__ */
+// Project version
+#define OAPV_VERSION_MAJOR @VERSION_MAJOR@
+#define OAPV_VERSION_MINOR @VERSION_MINOR@
+#define OAPV_VERSION_PATCH @VERSION_PATCH@
```

Comment:
Wasn't it decided that the header would only have the library version defines?

Reply:
> Wasn't it decided that the header would only have the library version defines?

At the moment, we still need this. The function oapv_version() returns the project version and requires the definitions OAPV_VERSION_MAJOR, OAPV_VERSION_MINOR, and OAPV_VERSION_PATCH. 
Currently, we still need two versions of the function. 
aopv_version() is used by the reference applications and returns the project version. 
The other is oapv_libversion(), which returns the library version and could be used for integration with other projects like FFmpeg. 

We will consider simplify it.

Reply:
Can't you add those defines in a different, also generated but not installed header?

Also, oapv_version() and oapv_libversion() should ideally return a const char*, or a strdup() output that the user needs to free otherwise.